pkg/agentconfig/container.go

```diff
@@ -216,3 +221,12 @@ func appendAppContainerEnvFrom(app *core.Container, cc *Container, es []core.Env
 	}
 	return es
 }
+
+func getIgnoredVolumeMounts(annotations map[string]string) map[string]bool {
+	vmSlice := strings.Split(annotations["telepresence.getambassador.io/inject-ignore-volume-mounts"], ",")
+	ivms := make(map[string]bool, len(vmSlice))
+	for _, vm := range vmSlice {
+		ivms[vm] = true
```

Comment:
```suggestion
		ivms[strings.TrimSpace(vm)] = true
```
because it's not unlikely that a user will use comma and space to separate the entries.